pages/radar.md

```diff
@@ -7,6 +7,9 @@ permalink: /radar.html
 
 [/home](/)
 
+## [Linux turns 30](https://twitter.com/linuxfoundation/status/1430539222142885898)
+25-August-2021: 30 years ago today Linus Torvalds introduced Linux to the world. Since then, Linux has become the leading operating system widely used in many applications, and in some, this is a de facto monopoly - say, [100% of supercomputers run Linux](https://itsfoss.com/linux-runs-top-supercomputers/). The operating system is constantly improving, new functions appear every release. Such active development has become possible because open licensing model. More than three-quarters of the new changes are committed by corporations. For example, Huawei Technologies [made 6.5% of all changes](https://lwn.net/Articles/860989/) for the latest stable kernel. What's the profit? The use in company products of this large collaborated development brings far more revenue than the cost of changes was made. The development of such a fully proprietary system is much more expensive and consequently would increase the final cost of products. It is clear that an open model of development is more competitive. The world runs on open source.
+
 ## [Huawei Release Their First RISC-V Development System – Hi3861](https://www.electropages.com/blog/2021/06/huawei-release-their-first-risc-v-development-system-hi3861)
 16-August-2021: Open software has many obvious advantages over proprietary. In general, a free license allows third-party developers - individual programmers and companies - to make their valuable contribution to the development of a product, and consequently, the result obtained has a chance to be more functional and competitive than the same proprietary one. However, open and free can be not only software but also hardware, such as processor architecture. If the [specification](https://riscv.org/technical/specifications/) of such an architecture is available for non-paid and free use, including commercial production of physical chips, such an architecture can as a result have [considerable support](https://riscv.org/members/) from end-producers and programmers, even if its current stage even lags behind the available proprietary solutions. We believe that using open technologies will lead to a more productive result than the development of own or the licensing of third-party proprietary hardware. Not to mention that such an approach is much less affected by the negative impact of unexpected cases such as international sanctions.
```

Comment:
16-August-2021: Open software has many apparent advantages over proprietary. In general, a free license allows third-party developers - individual programmers and companies - to make their valuable contributions to the development of a product. Consequently, the result obtained has a chance to be more functional and competitive than the same proprietary one. However, open and free can be not only software but also hardware, such as processor architecture. Suppose the [specification](https://riscv.org/technical/specifications/) of such an architecture is available on a non-paid and free use basis (including commercial production of physical chips). In that case, such an architecture can, as a result, have [considerable support](https://riscv.org/members/) from end-producers and programmers, even if its current stage even lags behind the available proprietary solutions. We believe that using open technologies will lead to a more effective result than developing own or licensing third-party proprietary hardware. Not to mention that such an approach is much less affected by the negative impact of unexpected cases such as international sanctions.

Reply:
Fixed